guacamole/src/main/frontend/src/app/client/templates/client.html

```diff
@@ -47,7 +47,7 @@
     
     <!-- Menu -->
     <div class="menu" ng-class="{open: menu.shown}" id="guac-menu">
-        <div class="menu-content" ng-if="menu.shown" guac-touch-drag="menuDrag">
+        <div class="menu-content" guac-touch-drag="menuDrag">
```

Comment:
It doesn't seem to be an issue anymore, no. I tried typing some accented keys into an SSH session and everything works as expected.

Reply:
I guess another question would be do these changes work if we _keep_ that `ng-if`?

I'm really apprehensive about removing it given that the commit mentions that this can happen, and keyboard handling is very finicky. I don't remember what was required to cause the problem before, but it's possible that the circumstances are subtle.

If this is truly necessary for these changes to work, then I think we'd need to:

1. Check out a copy of the code right after that `ng-if` was added via https://github.com/apache/guacamole-client/commit/9065b497c2333b6034a490ccfe41d36e91d21e5b.
2. Remove the `ng-if` (but do nothing else).
3. Confirm that we can reliably reproduce the problem noted by the commit message.
4. Switch back to the code from this PR.
5. Confirm that the same steps to reproduce the problem no longer reproduce the problem.

Reply:
Ok, I can't reproduce the issue described in GUACAMOLE-352 now, at https://github.com/apache/guacamole-client/commit/9065b497c2333b6034a490ccfe41d36e91d21e5b, or https://github.com/apache/guacamole-client/commit/9065b497c2333b6034a490ccfe41d36e91d21e5b with the `ng-if` removed. Any other ideas?